### PR TITLE
Fix frontmatter paths

### PR DESCRIPTION
**Pull request by lemonez**
No Jira ticket, this is to fix an issue caused where front matter paths didn't have a trailing forward slash.
- added forward slash to all front matter paths (`permalink`, `redirect_from`)
- alphabetized front matter for all the files I could tolerate in one sitting, plus a few

### CODE REVIEW

File: _pages/integrations/android-dl.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ layout: article
 title: "Android (Deep Learning)"
 categories: app-development
 tags: [integrations, app-development]
-permalink: integrations/android-dl
+permalink: integrations/android-dl/
```
Comment (warmbowski) [inline _pages/integrations/android-dl.md]: still some inconsistency with the leading slash on these paths.

Comment (warmbowski) [reply _pages/integrations/android-dl.md]: from your template PR
"- `permalink` and `redirect_from/to` properties have `/` at beginning and end (e.g., `/path/`)"

Comment (lemonez) [reply _pages/integrations/android-dl.md]: thanks fixed all the ones i could find, commit incoming

**Review by warmbowski — APPROVED** [review]
awesome!

**Pull request merged**